### PR TITLE
Use Codama v1.0.0

### PR DESCRIPTION
**Pull request by lorisleiva**
See https://github.com/codama-idl/codama/pull/234

### CODE REVIEW

File: clients/js/src/generated/instructions/store.ts [hunk 1]
```diff
@@ -111,17 +111,22 @@ export type StoreInput<TAccountConfigAccount extends string = string> = {
...
 };
 
-export function getStoreInstruction<TAccountConfigAccount extends string>(
-  input: StoreInput<TAccountConfigAccount>
+export function getStoreInstruction<
+  TAccountConfigAccount extends string,
+  TProgramAddress extends Address = typeof SOLANA_CONFIG_PROGRAM_ADDRESS,
+>(
+  input: StoreInput<TAccountConfigAccount>,
+  config?: { programAddress?: TProgramAddress }
```
Comment (buffalojoec) [inline clients/js/src/generated/instructions/store.ts:119]: Interesting, so one could use the config program client against a different config program (configurable program ID)?

File: program/idl.json [hunk 2]
```diff
@@ -150,6 +150,6 @@
     "origin": "shank"
   },
   "additionalPrograms": [],
-  "standard": "kinobi",
-  "version": "0.19.0"
+  "standard": "codama",
+  "version": "1.0.0"
```
Comment (buffalojoec) [inline program/idl.json:145]: _Major_ version ?? 👀

File: program/idl.json [hunk 3]
```diff
@@ -141,9 +95,46 @@
         "optionalAccountStrategy": "omitted"
       }
     ],
-    "definedTypes": [],
+    "definedTypes": [
+      {
+        "kind": "definedTypeNode",
+        "name": "configKeys",
```
Comment (buffalojoec) [inline program/idl.json:101]: Nice. I like how this is an IDL-defined type now, instead of in-lined.

**Review by buffalojoec — APPROVED** [review]
Looks great!!

File: clients/js/src/generated/instructions/store.ts (hunk 1 above)
Comment (lorisleiva) [reply clients/js/src/generated/instructions/store.ts:119]: Yes! This is a late addition from @mcintyre94 ([See PR](https://github.com/codama-idl/codama/pull/224)). For instance, this allows someone to use the SPL Token JS client for the Token 2022 program (if someone was taking bloody ages to release a Token 2022 client for example 👀).

File: program/idl.json (hunk 2 above)
Comment (lorisleiva) [reply program/idl.json:145]: Major version indeed! It's showtime. 🌟

File: clients/js/src/generated/instructions/store.ts (hunk 1 above)
Comment (buffalojoec) [reply clients/js/src/generated/instructions/store.ts:119]: I dig it!!

**Pull request merged**